File: node/dni.js
// Consulta DNI — GET /v1/dni/{numero}
const token = "<tu_token>";
const numero = "12345678";

const respuesta = await fetch(`https://mblapirest.com/api/v1/dni/${numero}`, {
  headers: { Authorization: `Bearer ${token}` },
});
console.log(await respuesta.json());

File: python/dni.py
"""Consulta DNI — GET /v1/dni/{numero}"""
import requests

token = "<tu_token>"
numero = "12345678"

respuesta = requests.get(
    f"https://mblapirest.com/api/v1/dni/{numero}",
    headers={"Authorization": f"Bearer {token}"},
)
print(respuesta.json())
